itou/metabase/tables/prolongation_requests.py

```diff
@@ -0,0 +1,39 @@
+from itou.approvals.enums import ProlongationRequestStatus
+from itou.approvals.models import ProlongationRequest
+from itou.metabase.tables.utils import (
+    MetabaseTable,
+    get_choice,
+    get_column_from_field,
+    get_common_prolongation_columns,
+)
+
+
+def get_field(name):
+    return ProlongationRequest._meta.get_field(name)
+
+
+TABLE = MetabaseTable(name="demandes_de_prolongation")
+TABLE.add_columns(
+    get_common_prolongation_columns(get_field_fn=get_field)
+    + [
+        {
+            "name": "id_prolongation",
+            "type": "integer",
+            "comment": "ID C1 de la prolongation",
+            # Trick to access a reverse OneToOneField without a related_name.
+            # See:
+            # - https://stackoverflow.com/questions/57881833/django-reverse-onetoonefield-matching-without-related-name
+            # - https://stackoverflow.com/questions/25944968/check-if-a-onetoone-relation-exists-in-django
+            "fn": lambda o: o.prolongation.pk if hasattr(o, "prolongation") else None,
```

Comment:
Je me demande si on ne peut pas tout betement mettre `prologation_id` ici ? Je pense que c'est comme ça que ils sont notés dans postgres, ça éviterait le hasattr.

Reply:
Eh non justement c'est pas si simple je me suis bien cassé les dents sur le reverse d'un OneToOneField. Le petit debugging ci-dessous devrait éclaircir les choses @vperron 
```
# sens normal - all right
ProlongationFactory().request_id  # None
ProlongationFactory().request  # None
ProlongationWithRequestFactory().request_id  # 1
ProlongationWithRequestFactory().request  # a ProlongationRequest object
# sens inverse - confusing
ProlongationRequestFactory().prolongation_id  # AttributeError
ProlongationRequestFactory().prolongation  # RelatedObjectDoesNotExist
ProlongationWithRequestFactory().request.prolongation_id  # AttributeError (big surprise here!)
ProlongationWithRequestFactory().request.prolongation  # a Prolongation object
```